.chronus/config.yaml

```diff
@@ -56,3 +56,7 @@ changelog: ["@chronus/github/changelog", { repo: "microsoft/typespec" }]
 
 ignore:
   - "@typespec/http-client-csharp"
+
+changedFiles:
```

Comment:
Should we include *.tsp under libs for our libraries?

Reply:
list is filtering out(with `!` prefix)